Signum.React/Scripts/Cookies.ts

```diff
@@ -1,7 +1,7 @@
 
 export namespace Cookies {
 
-  export function set(name: string, value: string, days?: number, path?: string) {
+  export function set(name: string, value: string, days?: number, path?: string, domain?: string) {

```

Comment:
domain parameter is not used in the body

Reply:
Thanks. Fixed in 36f239b11cd7c306026cdebfd75af841cac1d225.